### PR TITLE
app-admin/logstash-bin: bump to 5.6.3

### PR DESCRIPTION
**Pull request by hydrapolic**

### CODE REVIEW

File: app-admin/logstash-bin/logstash-bin-5.6.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+src_install() {
+	keepdir /etc/"${MY_PN}"/{conf.d,patterns,plugins}
+	keepdir "/var/lib/${MY_PN}"
+	keepdir "/var/log/${MY_PN}"
+
+	insinto "/usr/share/${MY_PN}"
+	newins "${FILESDIR}/agent.conf.sample" agent.conf
+
+	rm -v config/startup.options
```
Comment (monsieurp) [inline app-admin/logstash-bin/logstash-bin-5.6.3.ebuild]: missing `|| die` here.

File: app-admin/logstash-bin/logstash-bin-5.6.3.ebuild [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+# source: LICENSE.txt and NOTICE.txt
+LICENSE="Apache-2.0 MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RESTRICT="strip"
+QA_PREBUILT="opt/logstash/vendor/jruby/lib/jni/*/libjffi*.so"
+
+RDEPEND="virtual/jre:1.8"
```
Comment (monsieurp) [inline app-admin/logstash-bin/logstash-bin-5.6.3.ebuild]: You must `inherit java` otherwise it won't properly check for a JVM on the system.

**Comment by hydrapolic** [comment]
Thanks @monsieurp, pushed a new version.

**Comment by hydrapolic** [comment]
Thanks @monsieurp

**Pull request closed without merging**